test/indexer/data/fa5be5eb-2d64-49f5-8ed8-bd627ac9bc7a.dss.hca.json

```diff
@@ -604,7 +604,7 @@
             }
         },
         "project/627cb0ba-b8a1-405a-b58f-0add82c3d635": {
-            "describedBy": "https://schema.staging.data.humancellatlas.org/type/project/11.0.0/project",
+            "describedBy": "https://schema.staging.data.humancellatlas.org/type/project/19.0.0/project",
```

Comment:
This bundle is ancient. I asked for a recent bundle. Why did you pick this one?

To find the most recent bundle, use the schema version (`describedBy`) and the update date (`provenance.update_date`).

Reply:
> This bundle is ancient. I asked for a recent bundle. Why did you pick this one?

I had selected the `fa5be5eb` bundle since it allowed an existing test (`test_response_stage_projects_accessions`) to verify the new field's value in the response. The fixup commit adds the field to a more recent canned bundle instead, increasing the project entity's schema from `project/17.0.0` to `project/19.0.0`.

Reply:
It would help us both if you proactively communicated your reasoning when deviating from my review feedback.

> The fixup commit adds the field to a more recent canned bundle instead, increasing the project entity's schema from project/17.0.0 to project/19.0.0.

The bundle you picked now is involved in a test case in `test_response.py` that indexes during class setup so adding a test method there has minimal impact on the running time of this test. Please add a test method to that test that asserts the effects of this change on the hits and term facet part of the response. That test should make one request to every /index/{entity_type} endpoint with filter and sort parameter set to the field you are adding.